freqtrade/persistence.py

```diff
@@ -91,6 +91,7 @@ def check_migrate(engine) -> None:
         close_rate_requested = get_column_def(cols, 'close_rate_requested', 'null')
         stop_loss = get_column_def(cols, 'stop_loss', '0.0')
         initial_stop_loss = get_column_def(cols, 'initial_stop_loss', '0.0')
+        stoploss_order_id = get_column_def(cols, 'stoploss_order_id', 'null')
```

Comment:
there are 2 things missing with the migration:
* you need to add the newest column in line 85 of this script, so it detects that the database is not the latest
* add all columns apart from the new column to `test_migrate_new` (only `ticker_interval` should be missing) to test the migration works from the version of today/yesterday, to this latest verison

Reply:
nice ! will do

Reply:
yes